commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java

```diff
@@ -203,14 +203,11 @@ private String subValues(Object... arguments) {
 		return new StrSubstitutor(valueSub, "`(", ")`").replace(getMessage());
 	}
 
-	private static Pattern getPattern() {
-		if (replacePattern == null) {
-			replacePattern = Pattern.compile(VARIABLE_MARKER);
-		}
-		return replacePattern;
-	}
-
 	public static ErrorCode getByCode(int code) {
 		return CODE_TO_VALUE.get(code);
 	}
+
+	private static final class VariableMarker {
+		private static final Pattern REPLACE_PATTERN = Pattern.compile("`\\(([^()]*)\\)`");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use already-defined constant 'VARIABLE_MARKER' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)